test/gtest/common/test.cc

```diff
@@ -139,7 +139,7 @@ test_base::count_warns_logger(const char *file, unsigned line, const char *funct
         ++m_total_warnings;
     }
     pthread_mutex_unlock(&m_logger_mutex);
-    return UCS_LOG_FUNC_RC_CONTINUE;
+    return ucs_log_default_handler(file, line, function, level, comp_conf, message, ap);
```

Comment:
why needed? UCS_LOG_FUNC_RC_CONTINUE should achieve same effect no?

Reply:
yes, it is not needed, reverted